aspnetcore/security/authentication/google-logins.md

```diff
@@ -0,0 +1,138 @@
+---
```

Comment:
Done. I think it's helpful to mention each change that applies multiple times at most twice (second time at the end to cause the author to double-check, but not at every instance).